rest/tests/test_learning_resource.py

```diff
@@ -104,6 +107,34 @@ def test_missing_learning_resource(self):
                                    expected_status=HTTP_404_NOT_FOUND)
         self.get_learning_resource(repo_slug2, lr2_id)
 
+    def test_learning_resource_filter(self):
+        """Test for filtering learning resources by ids"""
+
+        def get_filtered(ids, expected_status=HTTP_200_OK):
+            """Return list of learning resourceswhich are in shopping cart"""
+            url_base = "{repo_base}{repo_slug}/learning_resources/".format(
+                repo_base=REPO_BASE,
+                repo_slug=self.repo.slug,
+            )
+            resp = self.client.get("{url_base}?id={ids}".format(
+                url_base=url_base,
+                ids=",".join([str(s) for s in ids])
+            ))
+            self.assertEqual(expected_status, resp.status_code)
+            if expected_status == HTTP_200_OK:
+                return [x['id'] for x in as_json(resp)['results']]
```

Comment:
Please return this list comprehension `sorted()` -- since it's being sorted multiple times in comparisons below.